### ISSUE TITLE
2D gray image for nuclei detection error

### ISSUE THREAD

**Issue by minjeejang** [MouseLand/cellpose#41]
Hi devs,

I've used this for my images, 2D, gray images of nuclei (1 channel) and got errors when running the following:
mask, flow, styles, diam = model.eval(image, diameter=30, channels=[0,0])

It worked well actually when I changed it as follows:
mask, flow, styles, diam = model.eval(image[None, ...], diameter=30, channels=[0,0])
which makes me think there seems to be a dimension error for such images. 

Thanks for the good tool! This is very useful.

**Comment by carsen-stringer** [comment]
Thanks for pointing this out, it wouldn't accept 2D images before (expected them in a list, e.g. `[image]`). I've added support for 2D inputs into `model.eval` now, but always make sure that the number of images processing is correct (now printed to command line). (`pip install cellpose --upgrade`)
